src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag

```diff
@@ -7,6 +7,9 @@
 
 <div class="well well-plain">
     <div>
+        <label class="label-control">Adding a Single Instructor</label>
+    </div><br/>
```

Comment:
Sorry, my mistake. All `<br/>` should be `<br>` according to our HTML coding style guide. https://docs.google.com/document/d/12PJYbQoqjK-0LzaUuguQ4kGE--eikCcHfwzZDGwFOJ0/pub?embedded=true